frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/create-local-volume-set.tsx

```diff
@@ -34,7 +35,7 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({ stat
       <LocalVolumeSetHeader />
       <div className="ceph-ocs-install__form-wrapper">
         <Form noValidate={false} className="ceph-ocs-install__create-sc-form">
-          <LocalVolumeSetInner state={state} dispatch={dispatch} />
+          <LocalVolumeSetInner state={state} dispatch={dispatch} isOCSView />
```

Comment:
Shouldn't this be a plugin based model where we inject things based on OCS view?